### ISSUE TITLE
A question about capacity of pre-trained model..

### ISSUE THREAD

**Issue by warrior13reg** [cisco/mindmeld#338]
Hi there,

I have a question for you and hope you will help me. 

Can the model be self-trained using the functions we make out? 

For example,

while I am talking to a bot I always use the same words, like "I think..", though, I can use other variants like "I feel or I am aware that" and so on. Is it possible to give a command to a bot to analyse the amount of repetitive words or phrases and then suggest me using other synonyms or phrases?

Thank you very much in advance.

**Comment by murali1996** [comment]
Hi @warrior13reg. I believe you are looking for some kind of data augmentation technique that automatically populates more paraphrased data from your initial/seed training set. If that's the case, Mindmeld now has an excellent data augmentation feature which you can leverage. To read more about this feature, you'll have to use the latest mindmeld as follows:

```
git clone https://github.com/cisco/mindmeld; cd mindmeld
pip install -r docs-requirements.txt
make apidoc
```
then proceed to section `Data Augmentation in MindMeld` in the compiled docs to read more.
